app/screens/index.tsx

```diff
@@ -191,6 +191,9 @@ Navigation.setLazyComponentRegistrator((screenName) => {
         case Screens.DISPLAY_SETTINGS:
             screen = withServerDatabase(require('@screens/settings/display').default);
             break;
+        case Screens.NOTIFICATION_SETTINGS_MENTION:
```

Comment:
nit and 0/5

Perhaps in order to keep the sorting and grouping of this we could do:
* SETTINGS
* SETTINGS_DISPLAY
* SETTINGS_NOTIFICATION
* SETTINGS_NOTIFICATION_MENTION

what do you think?